docs/reference/module-api/module-overview.md

```diff
@@ -2,29 +2,20 @@
 
 Module configuration objects may use the following configuration properties. The overall categories are broadly defined:
 - [Configuration settings](#configuration-settings): Static module settings. Once the module is initialized these settings are fixed and can't access the module itself or any other module's settings.
+- [Configuration cascades](#configuration-cascades): Settings that can either be static or set via functions that have access to the module itself.
```

Comment:
Practically speaking they have access to the options of the module itself. Access to the rest of `self` is technically there but there's not much attached to it yet.

Reply:
Changed.